ci/travis-install.sh

```diff
@@ -51,6 +51,5 @@ case "$TRAVIS_OS_NAME" in
 	;;
 esac
 
-echo "Python version: $(/usr/bin/python -V 2>&1)"
-echo "Python3 path: $(which python3 2>&1)"
+echo "Python3 path: $(type python3 2>&1)"
```

Comment:
`type python3` produces the output `python3 is /usr/bin/python3` and `type -p python3` produces the output `/usr/bin/python3`. Did you perhaps want to use `type -p`? Otherwise this patch series looks good to me.

Reply:
OK this has been fixed.